style.css

```diff
@@ -404,3 +404,148 @@ a.btn-see-resume:active {
   float: right;
 }
 
+.contact-form {
+  background-image: url(contact_form_shapes.png), url(Contact_me.png);
+  background-size: auto 50%;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.contact-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-info h2 {
+  font-size: 40px;
+  text-align: center;
+  color: white;
+  margin-top: 160px;
+  margin-bottom: 20px;
+}
+
+.contact-info p {
+  font-size: 20px;
+  text-align: center;
+  margin-bottom: 38px;
+  padding: 0 70px;
+  color: white;
+}
+
+.contact-input {
+  width: 87.2%;
+  height: 48px;
+  margin-bottom: 20px;
+  margin-left: 24px;
+  margin-right: 24px;
+  border-radius: 8px;
+  font-size: 17px;
+  border: none;
+  color: #172b4d;
+  text-indent: 10px;
+}
+
+.contact-textarea {
+  font-size: 17px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  border: none;
+  color: #172b4d;
+  text-indent: 10px;
+  padding-top: 10px;
+  margin-bottom: 10px;
+  width: 87.2%;
+  border-radius: 8px;
+  text-decoration: none;
+}
+
+.contact-textarea:focus {
+  border: 1px solid #6070ff;
+}
+
+.contact-button:hover {
+  background-color: #fff;
+  border: 1px solid white;
+  color: white;
+  font-size: 17px;
+}
+
+.input-container {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 8px;
+  margin-bottom: 20px;
+  margin-left: 24px;
+  margin-right: 24px;
+  align-self: flex-start;
+}
+
+.home-ind {
+  background-color: #0d151e;
+  border: none;
+  height: 4px;
+  border-radius: 4px;
+  width: 36%;
+}
+
+.btn-get-it-touch {
+  text-decoration: none;
+  color: #6070ff;
+  border: 1px solid #396df2;
+  padding: 7px;
+  line-height: 30px;
+  font-size: 17px;
+  border-radius: 8px;
+  margin-bottom: 20px;
+  background-color: #fff;
+}
+
+.btn-get-it-touch:disabled {
+  font-weight: 300;
+  font-size: 17px;
+  font-family: 'Poppins', sans-serif;
+  background: #ebebff;
+  color: #808080;
+  border: 2px solid #808080;
+  border-radius: 4px;
+  padding: 7px;
+}
+
+a.btn-get-it-touch:hover {
+  border: 1px solid #396df2;
+  padding: 7px;
+  border-radius: 4px;
+  font-size: 17px;
+  font-weight: 300;
+  background: #396df2;
+  color: #ebebff;
+  font-family: 'Poppins', sans-serif;
+}
+
+a.btn-get-it-touch:active {
+  padding: 7px;
+  border-radius: 4px;
+  font-size: 17px;
+  font-weight: 300;
+  background: #000080;
+  color: #ebebff;
+  font-family: 'Poppins', sans-serif;
+}
```

Comment:
- [ ] Great job implementing stylings for all button actions, but as requested, you should follow the Figma design. Did you know that the designs usually have a buttons-only section? You can check your here:

![image](https://user-images.githubusercontent.com/66967274/144044101-520b333e-abd2-45cb-95c4-4da1f3678241.png)